### DISCUSSION TITLE
account suspension without any notification

### DISCUSSION THREAD

**Discussion by mikhaylovia** [community/community#51033] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My account 'fr0st61te' got suspended on 1 march without any notifications. I've wrote 2 tickets:
1. on 6 march when I've noticed this incident, ticket id: 2044003
2. on 15 march to remind about my problem, ticket id: 2060640

I still don't have any answers about block in mail nor in spam. I can provide any information about me if it's needed, like photo id, legal name or etc. I don't know but it may depends on place where I worked before due to sanctions possibilities, I could leave my old work mail in profile.

**Comment by tmpyc0304** [comment]
I already wait for a month and haven't get any receive. Is there anyone will read my ticket?

**Reply by mikhaylovia** [reply]
How many tickets did you write to them? I'm sorry to disturb @queenofcorgis, maybe you can help us?

**Reply by tmpyc0304** [reply]
at least 10.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 

I checked on your ticket and it is in the right place. Tickets are reviewed in the order they were received and response time is based on our team's resources. The Community is unable to assist in account-specific matters like these nor can we provide escalation so I'm going to go ahead and lock this post. Thank you for your patience!
